spec/acceptance/nginx_prometheus_exporter_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper_acceptance'
+
+describe 'prometheus nginx exporter' do
+  it 'nginx_prometheus_exporter works idempotently with no errors' do
+    on hosts, puppet('module', 'install', 'puppet-nginx', '--version', '3.2.1')
```

Comment:
can you use the example from the grafana module I linked? That's better because it doesn't hardcode a specific version

Reply:
done